client/src/containers/SudoMode/SudoMode.js

```diff
@@ -35,12 +113,83 @@ const withSudoMode = (WrappedComponent) => {
       return this.state.active === true;
     }
 
+    /**
+     * Renders a notice to the user that they will need to verify themself
+     * to enter sudo mode and continue to use this functionality.
+     *
+     * @returns {HTMLElement}
+     */
     renderSudoModeNotice() {
-      return <p>Sudo mode notice</p>;
+      const { i18n } = window;
+      const { showVerification } = this.state;
+
+      const helpLink = Config.getSection(configSectionKey).helpLink || null;
+
+      return (
+        <div className="sudo-mode__notice sudo-mode__notice--required">
+          <p className="sudo-mode__notice-message">
+            { i18n._t('SudoMode.VERIFY_ITS_YOU', 'Verify it\'s you first.') }
+            { helpLink && (
+              <a href={helpLink} className="sudo-mode__notice-help" target="_blank" rel="noopener noreferrer">
+                { i18n._t('SudoMode.HELP_TEXT', 'What is this?') }
+              </a>
+            ) }
+          </p>
+          { !showVerification && (
+            <Button
+              className="sudo-mode__notice-button font-icon-lock"
+              color="info"
+              onClick={this.handleConfirmNotice}
+            >
+              { i18n._t('SudoMode.VERIFY', 'Verify to continue') }
+            </Button>
+          ) }
+        </div>
+      );
     }
 
+    /**
+     * Renders the password verification form to enter sudo mode
+     *
+     * @returns {HTMLElement}
+     */
     renderSudoModeVerification() {
-      return <p>Sudo mode verification</p>;
+      const { i18n } = window;
+      const { errorMessage } = this.state;
+
+      const inputProps = {
+        type: 'password',
+        name: 'sudoModePassword',
+        id: 'sudoModePassword',
+        className: 'no-change-track',
+        onKeyPress: this.handleVerifyInputKeyPress,
+        innerRef: (node) => this.setPasswordInput(node),
```

Comment:
I think this can just be:

```suggestion
        innerRef: this.setPasswordInput,
```